pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
I'm not using CMake, ported `vcprox` and `sln` files to gyp / `xcodeproj` 

BTW; IMO this issue is NOT whether BIT64 is defined or not.

If `BIT64` refers to `x86` and `ARM` ?? I will update PR to cover `_Is_integral` duplication issue only. Otherwise this PR should be fine?